main.go

```diff
@@ -69,33 +70,43 @@ func main() {
 	}
 
 	for {
+		onlyDelimitCheck := false
 		content, readAllErr := buf.ReadBytes('\n')
 		if readAllErr != nil && readAllErr != io.EOF {
 			fmt.Fprintln(os.Stderr, readAllErr)
 			os.Exit(1)
 		}
 
 		contentLineSplit := bytes.Split(content, []byte{'\n'})
-		data = bytes.Split(contentLineSplit[0], delimiter)
 
-		lineLength := len(data)
-		var output [][]byte
+		if *onlyDelimitParameter {
+			if bytes.Contains(contentLineSplit[0], delimiter) {
+				onlyDelimitCheck = true
+			}
+		}
+
+		if onlyDelimitCheck {
```

Comment:
Any way to shorten this to a one-liner?

Reply:
if bytes.Contains(contentLineSplit[0], delimiter) && *onlyDelimitParameter {
}

Reply:
Exactly.

Reply:
but if i want to just use one parameter later on like e.g -f, -c -> *onlyDelimitParameter will be always false and so on the program wont work

Reply:
But then your condition was already wrong and this has nothing to do with the shortening I requested.

Reply:
u may be right with that